Brewfile

```diff
@@ -10,3 +10,4 @@ brew "openal-soft"
 brew "mbedtls"
 brew "libuv"
 brew "openssl"
+brew "unzip"
```

Comment:
Is this really necessary? `unzip` is a built-in command on OS X.

Reply:
Yeah, probably not. I'm using Windows and haven't tested it on a Macbook yet.